components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.html

```diff
@@ -10,18 +10,20 @@
   </span>
   <chef-icon aria-hidden>keyboard_arrow_down</chef-icon>
 </button>
-<chef-dropdown class="dropdown" *ngIf="dropdownActive" (keydown.esc)="handleEscape()">
```

Comment:
Most of this is just changing order of the markup so the `chef-click-outside` element wraps around the `chef-dropdown` element (which is generally the convention we use for using the `chef-click-outside` element. 

Also removes an unnecessary argument on line 25 where `handleApplyClick()` had an unnecessary `$event` argument that was throwing errors in CI.